src/imitation/scripts/common/expert.py

```diff
@@ -43,3 +43,19 @@ def config_hook(config, command_name, logger):
 @expert_ingredient.capture
 def get_expert_policy(venv, policy_type, loader_kwargs, common):
     return serialize.load_policy(policy_type, venv, **loader_kwargs)
+
+
+@expert_ingredient.capture
+def download_expert_rollouts(rollout_path, loader_kwargs):
+    if not rollout_path.endswith("-huggingface"):
+        raise ValueError(
+            "`rollout_path` must follow the convention `{algo}-huggingface`"
+            "to download rollouts from huggingface of an expert trained using {algo}."
+            "Example: demonstrations.rollout_path=ppo-huggingface",
+        )
+    algo_name = rollout_path.split("-")[0]
```

Comment:
This will fail if the path has multiple dashes. You should use regex or pathlib instead